core/service/validation-service.js

```diff
@@ -0,0 +1,81 @@
+var Montage = require("montage").Montage,
+    Services = require("core/model/services").Services,
+    FastMap = require("collections/fast-map"),
+    FastSet = require("collections/fast-set");
+
+exports.ValidationService = Montage.specialize({
+    ACTIONS: {
```

Comment:
I know there is just one validation service within the app but that could be set on the constructor :)